templates/select-options.php

```diff
@@ -0,0 +1,134 @@
+	$generator = sprintf( __( 'WordPress %s', 'wordpress-importer' ), $matches[1] );
+}
+
+?>
+<div class="welcome-panel">
+	<div class="welcome-panel-content">
+		<h2><?php esc_html_e( 'Step 2: Import Settings', 'wordpress-importer' ) ?></h2>
+		<p><?php esc_html_e( 'Your import is almost ready to go. Before your content is imported, pick exactly how you want your data imported.', 'wordpress-importer' ) ?></p>
+
+		<div class="welcome-panel-column-container">
+			<div class="welcome-panel-column">
+				<h3><?php esc_html_e( 'Import Summary', 'wordpress-importer' ) ?></h3>
+				<ul>
+					<li>
+						<span class="dashicons dashicons-admin-post"></span>
+						<?php echo esc_html( sprintf(
+							_n( '%d post (including CPTs)', '%d posts (including CPTs)', $data->post_count, 'wordpress-importer' ),
+							$data->post_count
+						)) ?>
+					</li>
+					<li>
+						<span class="dashicons dashicons-admin-media"></span>
+						<?php echo esc_html( sprintf(
+							_n( '%d media item', '%d media items', $data->media_count, 'wordpress-importer' ),
+							$data->media_count
+						)) ?>
+					</li>
+					<li>
+						<span class="dashicons dashicons-admin-users"></span>
+						<?php echo esc_html( sprintf(
+							_n( '%d user', '%d users', count( $data->users ), 'wordpress-importer' ),
+							count( $data->users )
+						)) ?>
+					</li>
+					<li>
+						<span class="dashicons dashicons-admin-comments"></span>
+						<?php echo esc_html( sprintf(
+							_n( '%d comment', '%d comments', $data->comment_count, 'wordpress-importer' ),
+							$data->comment_count
+						)) ?>
+					</li>
+					<li>
+						<span class="dashicons dashicons-category"></span>
+						<?php echo esc_html( sprintf(
+							_n( '%d term', '%d terms', $data->term_count, 'wordpress-importer' ),
+							$data->term_count
+						)) ?>
+					</li>
+				</ul>
+			</div>
+			<div class="welcome-panel-column">
+				<h3><?php esc_html_e( 'Facts About Your Import', 'wordpress-importer' ) ?></h3>
+				<ul>
+					<li>
+						<?php echo wp_kses( sprintf(
+							__( 'Exported from <a href="%1$s">%2$s</a>', 'wordpress-importer' ),
+							esc_url( $data->home ),
+							esc_html( $data->title )
+						), 'data' ) ?>
+					</li>
+					<li>
+						<?php echo esc_html( sprintf(
+							__( 'Generated by %s', 'wordpress-importer' ),
+							$generator
+						)) ?>
+					</li>
+					<li>
+						<?php echo esc_html( sprintf(
+							__( 'Format: WXR v%s', 'wordpress-importer' ),
+							$data->version
+						)) ?>
+					</li>
+				</ul>
+			</div>
+			<div class="welcome-panel-column">
+				<h3><?php esc_html_e( 'Facts About the Sea Lion', 'wordpress-importer' ) ?></h3>
+				<ul>
+					<li><?php esc_html_e( 'Unlike seals, sea lions have external ear flaps.', 'wordpress-importer' ) ?></li>
+					<li><?php esc_html_e( 'Sea lions are carnivorous and primarily eat fish and squid.', 'wordpress-importer' ) ?></li>
```

Comment:
Moar of these. Or maybe some random animal. I fee like I don't know enough about manatees